config/site/site.config.js

```diff
@@ -2,7 +2,7 @@ const i18nSupportedLocales = require('terra-i18n/lib/i18nSupportedLocales');
 const navConfig = require('./navigation.config');
 const startCase = require('lodash.startcase');
 const fs = require('fs');
-const path = require('path');
+const path = require('path').posix;
 
 const npmPackage = JSON.parse(fs.readFileSync(path.resolve(process.cwd(), 'package.json')));
```

Comment:
On Windows the filepath gets resolved to "C:\terra-core\C:\terra-core\package.json".

I believe this because since path is now in posix its looking for "C:/terra-core/package.json"

Reply:
From my understanding of it I thinking we can just change the line to 
`const npmPackage = JSON.parse(fs.readFileSync(path.resolve('package.json')));`
since the last thing path.resolve tries to do is add the current working directory to resolve it.

Reply:
Lets change it to path.join.